src/main.js

```diff
@@ -362,31 +362,8 @@ function TimekitBooking() {
     var newConfig = {};
     var localizationConfig = {};
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['localizationConfig' is defined but never used.](https://www.codacy.com/app/laander/booking-js/file/1372227308/issues/source?bid=2969108&fileBranchId=2977269#l363)